TF_Application/Assets/TouchFree/Tooling/Scripts/Connection/MessageReceiver.cs

```diff
@@ -132,14 +132,19 @@ void HandleConfigState(ConfigState _configState)
             }
         }
 
+        Vector2 lastKnownCursorPosition = new Vector2();
```

Comment:
This should probably be declared at the top with other properties. Also lacks documenting comments